conda/install.py

```diff
@@ -397,23 +405,49 @@ def replace_long_shebang(mode, data):
     return data
 
 
+def replace_entry_point_shebang(all_data, placeholder, new_prefix):
+    """Code adapted from pyzzer.  This is meant to deal with entry point exe's created by distlib, which
```

Comment:
How literally is it adapted? If code is copied, don't forget to include the copyright notice:

https://bitbucket.org/vinay.sajip/pyzzer/src/5d5740cb04308f067d5844a56fbe91e7a27efccc/LICENSE?at=default&fileviewer=file-view-default